.github/workflows/integrationTestsWindows.yml

```diff
@@ -21,7 +21,6 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
-      - run: npm config set scripts-prepend-node-path true
```

Comment:
Why are we getting rid of this?

Reply:
When running windows integration tests, the npm command is failing stating the option `scripts-prepend-node-path` is not supported. I assume that the later version of npm have removed this setting.